Jenkinsfile

```diff
@@ -292,7 +289,7 @@ def apiTest (String STAGE_NAME, String BASE_URL, String ENV_SUFFIX, Boolean run_
                                 -r cli,junit,html
                         """
 
-                        if (run_fixture_tests) {
+                        if ("dev".equalsIgnoreCase("${ENV_SUFFIX}")) {
```

Comment:
General comment, related to my hate of the "dev" branch, no change required:
This could get confusing.
So when we're in "dev" the branch, we don't want newman tests to run. But when there's a "dev" appended to the name we do want it to run. But when "dev" is appended, we're not on the "dev" branch. Grrr.

Reply:
Let's suggest calling the branch 'release' or something else at tomorrow's meeting.